client/src/pages/Settings/Settings.tsx

```diff
@@ -42,9 +41,9 @@ const TabPanel = (props: TabPanelProps) => {
 
 const Settings = (): JSX.Element => {
   const classes = useStyles();
-  const [currentTab, setCurrentTab] = useState('');
   const [currentTabIndex, setCurrentTabIndex] = useState<number>(0);
-  const handleTabIndexChange = (event: any, newValue: number) => {
+  // eslint-disable-next-line
+  const handleTabIndexChange = (event: any, newValue: number): void => {

```

Comment:
I am going to assume you are disabling the 'no-implicit-any' warning here. I would suggest to add a type to that event and remove the eslint warning that way.

Reply:
The reason I didn't specify type of this is because this is backlog [issue](https://github.com/mui-org/material-ui/issues/17454) with material-ui. However, I guess I can specify this type `ChangeEvent<{}>` but then this type gives me these errors:
```
  Line 46:52:  Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead  @typescript-eslint/ban-types
```

Reply:
I would take a look at where you are firing the event and it generally gives you an idea of what the type should look like. Event types are tricky in React, I tend to just start at the source of where I am firing this event and then copy what I see in my editor.

Reply:
Have you seen the MUI contributors comment [here?](https://github.com/mui-org/material-ui/issues/17454#issuecomment-648187083)

Reply:
Actually, I have tried the `SyntheticEvent` but it doesn't work. But following your advise, I looked at the console.log of event and found out that the event type is `BaseSyntheticEvent`. Thank you for your help.